lab_3_ann_retriever/main.py

```diff
@@ -105,6 +106,14 @@ def tokenize(self, text: str) -> list[str] | None:
         In case of corrupt input arguments, None is returned.
         """
 
+        if any((not isinstance(text, str), not text)):
```

Comment:
What does `not text` mean in this line?

Reply:
it's supposed to be a check for None. 
I'd change it to "text is None"